### ISSUE TITLE
Android App Sleep timer not working

### ISSUE THREAD

**Issue by justinwoodland** [advplyr/audiobookshelf-app#140]
### Steps to reproduce

1. Play an audiobook
2. Press the sleep timer button

### Expected behaviour
Sleep timer starts, counts down until 0 and stops playback.

### Actual behaviour
Playback stops immediately and no countdown started.

### Environment data

Audiobookshelf Version: 
Android App version: version 0.9.41-beta
Server version: 2.0.2

#### Android Issue
Android version:
11
Device model:
OnePlus 7T
Stock or customized system:
Stock

**Comment by advplyr** [comment]
I'm not able to replicate this, have you tried with different audiobooks?

**Comment by justinwoodland** [comment]
I have tried it with several different audiobooks. I've also noticed that after playback stops and I shake my phone (stumbled upon the feature to reset the timer before this issue occurred, but great feature) I hear a fraction of a second of playback and then it stops again, like the timer is being set to a very short interval.

I've collected the following from logcat while enabling a timer:

```
2022-04-24 22:29:53.036 28662-28662/? D/SleepTimerManager: Setting Sleep Timer for 900000 is chapter time false
2022-04-24 22:29:53.037 28662-28662/? D/SleepTimerManager: Timer Elapsed 1000 | Sleep TIMER time remaining 0 s
2022-04-24 22:29:53.037 28662-28662/? D/SleepTimerManager: Sleep Timer Pausing Player on Chapter
```

**Comment by heikomat** [comment]
I too have the same problem.
The playback immediately stops.

yesterday evening, when setting the sleeptimer to 90 minutes it showed as something about **negative** 7000 seconds.
But most of the time playback just instantly stops.
Some minutes later choosing any sleetimer-value set the timer 5 minutes, no matter what i chose.

I'm on a Pixel 3 with Androiod 12. The bug seemed to have appeared in app version 0.9.41, but i might have missed a version.
Updating to the current 0.9.42 didn't fix it

**Comment by advplyr** [comment]
Thanks for the logs. It will be fixed in the next release.

**Comment by advplyr** [comment]
Fixed in [v0.9.43-beta](https://github.com/advplyr/audiobookshelf-app/releases/tag/v0.9.43-beta)
